src/backend/joanie/core/models/products.py

```diff
@@ -379,6 +381,28 @@ class OrderGroup(BaseModel):
         on_delete=models.CASCADE,
     )
     is_active = models.BooleanField(_("is active"), default=True)
+    # Available start to end period of activation of the OrderGroup
+    start = models.DateTimeField(
+        help_text=_("order group’s start date and time"),
```

Comment:
Maybe something like this ?
```suggestion
        help_text=_("Date at which the order group activation begins"),
```